docs/appendix/zwe_server_command_reference/zwe/init/zwe-init-vsam.md

```diff
@@ -80,6 +80,8 @@ Full name|Alias|Type|Required|Help message
 
 Error code|Exit code|Error message
 |---|---|---
+ZWEL0148E|148|%s (%s) is not defined in Zowe YAML configuration file.
```

Comment:
Are both of the problems really about the definition in Zowe YAML? 
Is there the same guide to fix the issue?